web/next.config.js

```diff
@@ -309,6 +309,108 @@ const nextConfig = {
+			},
+			{
+				source:
+					"/about/nice-communities/library-and-knowledge-services-staff/buy-books-journals-and-databases",
+				destination:
+					"/library-and-knowledge-services/buy-books-journals-and-databases",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/nice-communities/library-and-knowledge-services-staff/buy-books--journals-and-databases/organisations-eligible-to-use-the-framework",
+				destination:
+					"/library-and-knowledge-services/buy-books-journals-and-databases/organisations-eligible-to-use-the-framework",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/nice-communities/library-and-knowledge-services-staff/buy-books-journals-and-databases/provider-information",
+				destination:
+					"/library-and-knowledge-services/buy-books-journals-and-databases/provider-information",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/nice-communities/library-and-knowledge-services-staff/buy-books--journals-and-databases/provider-information/sole-supplied-titles",
+				destination:
+					"/library-and-knowledge-services/buy-books-journals-and-databases/sole-supplied-titles",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/nice-communities/library-and-knowledge-services-staff/buy-books--journals-and-databases/purchasing-steps--direct-awards",
+				destination:
+					"/library-and-knowledge-services/buy-books-journals-and-databases/purchasing-steps-direct-awards",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/nice-communities/library-and-knowledge-services-staff/buy-books--journals-and-databases/purchasing-steps--further-competition",
+				destination:
+					"/library-and-knowledge-services/buy-books-journals-and-databases/purchasing-steps-further-competition",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/what-we-do/evidence-services/journals-and-databases/OpenAthens",
+				destination: "/library-and-knowledge-services/openathens",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/what-we-do/evidence-services/journals-and-databases/openathens/openathens-support",
+				destination:
+					"/library-and-knowledge-services/openathens/openathens-support",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/what-we-do/evidence-services/journals-and-databases/openathens/openathens-eligibility",
+				destination:
+					"/library-and-knowledge-services/openathens/openathens-eligibility",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/what-we-do/evidence-services/journals-and-databases/openathens/openathens-registration-help",
+				destination:
+					"/library-and-knowledge-services/openathens/openathens-registration-help",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/what-we-do/evidence-services/journals-and-databases/openathens/access",
+				destination:
+					"/library-and-knowledge-services/openathens/openathens-access",
+				permanent: true,
+			},
+			{
+				source: "/get-involved/jobs",
```

Comment:
Liza has just flagged on the Varnish that `/get-involved/jobs` redirects to `/get-involved/careers`, so seems like we need a rule for `get-involved/careers` too please (https://nicedigital.atlassian.net/browse/NXT-643?focusedCommentId=79526)